File: CAMPUSMONK/L5_find_minANDmax.py
num=int(input("array size->"))
#--------
def min(a,b):
    if a>b:
        return a
    else:
        return b
#-------------
arr=[]
for i in range(0,num):
    arr.append(int(input("enter number =")))
#-----
mx=arr[0]
#-----
# for i in range(0,num):
#     if arr[i]>mx:
#         mx=arr[i]
# print("maximum number is ==>",mx)
#---------------------------------------
# print("maximum number is ==>",max(arr))    
#-------------------------------
#chreat minimum function
for i in range(0,num):
    mn=min(mx,arr[i])
print(mn) 

File: CAMPUSMONK/L5_find_minANDmax.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    int n; cin>>n;
    int arr[n];
    for(int i=0;i<n;i++)
    {
        cin>>arr[i];
    }
    int mx=arr[0];
    for(int i=0;i<n;i++)
    {
        // if(arr[i]>mx)
        // {
        //     mx=arr[i];
        // }
        mx = max(mx,arr[i]);
    }
    cout<<mx;
}